### PR TITLE
Update rubygem-katello spec

### PR DESCRIPTION
**Pull request by ekohl**
This is based on gem2rpm/foreman_plugin.spec.erb with some local changes.

### CODE REVIEW

**Comment by ekohl** [comment]
Looks like the build failed because we need a Foreman RPM with https://github.com/theforeman/foreman-packaging/commit/aaad834c05995a8b0c99143b7ef702af567408ed included

**Comment by ekohl** [comment]
Depends on https://github.com/theforeman/foreman-packaging/pull/2435

**Comment by ekohl** [comment]
Updated. If this works, I'll update the foreman_plugin template so plugins with webpack should be fixed out of the box.

**Comment by ekohl** [comment]
[test rpm]

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 1]
```diff
@@ -1,121 +1,228 @@
...
 %{foreman_apipie_cache_plugin}
 %{foreman_assets_plugin}
-%{gem_instdir}/public/assets/bastion_katello
-
-%exclude %{gem_cache}
-%exclude %{gem_instdir}/package.json
-%exclude %{gem_instdir}/webpack
-
-%license %{gem_instdir}/LICENSE.txt
+%{foreman_webpack_plugin}
```
Comment (ehelms) [inline packages/katello/rubygem-katello/rubygem-katello.spec]: Also needs `%{foreman_webpack_foreman}`

**Comment by ekohl** [comment]
[test rpm]

**Comment by ekohl** [comment]
Should be rebased on https://github.com/theforeman/foreman-packaging/pull/2438 once that's merged.

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 2]
```diff
@@ -1,121 +1,229 @@
...
 %{gem_instdir}/vendor
+%exclude %{gem_cache}
+%exclude %{gem_instdir}/package.json
+%exclude %{gem_instdir}/webpack
 %{gem_spec}
 %{foreman_bundlerd_plugin}
 %{foreman_apipie_cache_foreman}
 %{foreman_apipie_cache_plugin}
 %{foreman_assets_plugin}
-%{gem_instdir}/public/assets/bastion_katello
```
Comment (ehelms) [inline packages/katello/rubygem-katello/rubygem-katello.spec]: This is still needed and the cause of the current failure

**Pull request merged**